utils.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Apr 16 19:12:51 2023
+
+@author: Sumedh Prasad
+
+utils code
+"""
+
+from typing import List, Tuple
+import os
+from os.path import join, dirname
+import logging
+
+from pathlib import Path
+import mysql.connector as mysql
+from dotenv import load_dotenv,dotenv_values
```

Comment:
line 17 - no need to import `dotenv_values`

Reply:
removed dotenv_values. Completed these steps.